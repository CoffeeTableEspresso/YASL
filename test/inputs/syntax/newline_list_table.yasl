
let x = {
    .a: 10,
    .b: 11
}

echo x

let y = [
    1,
    2
]

echo y
