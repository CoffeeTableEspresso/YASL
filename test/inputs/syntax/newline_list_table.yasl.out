{a: 10, b: 11}
[1, 2]
